diff --git a/maniflow/workspace/train_maniflow_robotwin_workspace.py b/maniflow/workspace/train_maniflow_robotwin_workspace.py
--- a/maniflow/workspace/train_maniflow_robotwin_workspace.py
+++ b/maniflow/workspace/train_maniflow_robotwin_workspace.py
@@ -45 +45 @@
-from maniflow.policy.maniflow_pointcloud_policy import ManiFlowTransformerPointcloudPolicy
+from maniflow.policy.maniflow_image_policy import ManiFlowTransformerImagePolicy
@@ -71 +71 @@
-        self.model: ManiFlowTransformerPointcloudPolicy = hydra.utils.instantiate(cfg.policy)
+        self.model: ManiFlowTransformerImagePolicy = hydra.utils.instantiate(cfg.policy)
@@ -73 +73 @@
-        self.ema_model: ManiFlowTransformerPointcloudPolicy = None
+        self.ema_model: ManiFlowTransformerImagePolicy = None
